Migration headsup (#1204)
microsoft/FLAML: README.md

    <br>
</p>

> :fire: Heads-up: We're excited to inform the community that we will be migrating [autogen](https://microsoft.github.io/FLAML/docs/Use-Cases/Autogen) into a dedicated github repository. Alongside this move, we'll also be launching a dedicated Discord server and a website for comprehensive documentation. The migration is expected to happen before Sep 18, 2023.

Reviewer: How about removing the date?

Author: Sure. Removed.

    <br>
</p>

> :fire: Heads-up: We're excited to inform the community that we will be migrating [autogen](https://microsoft.github.io/FLAML/docs/Use-Cases/Autogen) into a dedicated github repository. Alongside this move, we'll also be launching a dedicated Discord server and a website for comprehensive documentation.

Reviewer: ```suggestion
:fire: Heads-up: We're preparing to migrate [autogen](https://microsoft.github.io/FLAML/docs/Use-Cases/Autogen) into a dedicated github repository. Alongside this move, we'll also launch a dedicated Discord server and a website for comprehensive documentation.
```
